components/style/properties/gecko.mako.rs

```diff
@@ -3723,14 +3723,14 @@ fn static_assert() {
                 }
 
                 if let Some(bottom) = rect.bottom {
-                    self.gecko.mClip.height = bottom.0 - self.gecko.mClip.y;
+                    self.gecko.mClip.height = bottom.0.saturating_sub(self.gecko.mClip.y);
```

Comment:
Why not `(bottom - Au(self.gecko.mClip.y)).0`? Seems more consistent if we change the the behavior of `Au`.